egs/wsj/s5/steps/nnet3/components.py

```diff
@@ -429,14 +433,14 @@ def AddBLstmLayer(config_lines,
     assert(len(lstm_delay) == 2 and lstm_delay[0] < 0 and lstm_delay[1] > 0)
     output_forward = AddLstmLayer(config_lines, "{0}_forward".format(name), input, cell_dim,
                                   recurrent_projection_dim, non_recurrent_projection_dim,
-                                  clipping_threshold, norm_based_clipping,
```

Comment:
You can remove 'norm_based_clipping' option at this level and say in the usage message that it's deprecated.
Also reword the usage message for --clipping-threshold to say BackpropTruncationComponent.